src/components/dashboard/KeyPerformanceIndicator.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div class="kpi-container">
+    <div class="centered">
+      <span class="has-text-weight-bold is-size-1 has-text-centered">
+        {{ value }}
+      </span>
+      <p class="has-text-centered has-text-weight-bold">
+        {{ title }}
+      </p>
+    </div>
+  </div>
+</template>
+
+<script lang="ts" setup>
+defineProps<{
+  title: string;
+  value: number | string;
+}>();
+</script>
+
+<style scoped>
+.kpi-container {
+  gap: 1.5rem;
```

Comment:
it looks weird to me to have a gap on this container when it's not obviously in a gap or flex - should some styles move around so this looks more typical?

Reply:
I updated this style !